backend/src/cusco/sources/gleif.py

```
@@ -114,3 +118,110 @@ def _parse_record(self, raw: dict) -> LEIRecord:
+                    f"{GLEIF_API_BASE}/lei-records/{lei}/direct-parent"
+                )
+                if resp.status_code == 200:
+                    data = resp.json()
+                    raw = data.get("data")
+                    if raw:
+                        parent = self._parse_group_member(raw, "parent")
+                elif resp.status_code != 404:
+                    logger.warning(
+                        f"GLEIF direct-parent returned {resp.status_code} for {lei}"
+                    )
+            except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)